app/controllers/spree/admin/taxjar_backfills_controller.rb

```diff
@@ -2,9 +2,9 @@ module Spree
   module Admin
     class TaxjarBackfillsController < Spree::Admin::BaseController
       def create
-        ::SuperGood::SolidusTaxjar::BackfillTransactions.new.call
-        flash[:success] = "Transactions Backfilled!"
-        redirect_back(fallback_location: edit_admin_taxjar_settings_path)
+        order_numbers_backfilled = ::SuperGood::SolidusTaxjar::BackfillTransactions.new.call
+        @logs = "Transactions Backfilled! #{order_numbers_backfilled.map { |number| "#{number} successfully backfilled." }.join(" ")}"
```

Comment:
I know this is just a first draft, but I think we can make this messaging a little nicer. Maybe something like:
```ruby
"Successfully backfilled transactions for #{order_numbers_backfilled.count} orders. (#{order_numbers_backfilled.join(", ")})"
```

No need to print out `successfully backfilled` for every single order.

Reply:
Addressed!